README.md

```diff
@@ -445,13 +441,21 @@ I would love to receive contributions to this project and welcome PRs from every
 
 ## FAQ
 
-- __My terminal window does not close on selection, even when using `clipse -fc $PPID`__ - _Some terminal environments reference system variables differently. For example, the fish terminal will need to use `$fish_pid` instead. To debug this error you can run `echo $PPID` to see what gets returned. The 'close on selection functionality is also not currently available for macOS as killing the terminals ppid does not close the window - it seems AppleScript is needed to achieve this._
+- __My terminal window does not close on selection, even when using `clipse -fc $PPID`__ 
+<br>
+_Some terminal environments reference system variables differently. For example, the fish terminal will need to use `$fish_pid` instead. To debug this error you can run `echo $PPID` to see what gets returned. The 'close on selection functionality is also not currently available for macOS as killing the terminals ppid does not close the window - it seems AppleScript is needed to achieve this._
```

Comment:
I don't get the

`The 'close` 

I mean the quote

Reply:
missing closing quote, meant to be "close on select"